### ISSUE TITLE
Add Carbon Marketing Theme

### ISSUE THREAD

**Issue by jeffchew** [carbon-design-system/ibm-dotcom-library#285]
_hahnrob created the following on Jul 17:_  

 #### User Story
> As a `[user role below]`:
IBM.com Library Adopter

> I need to:
apply an expressive theme over Carbon components used in my application

> so that I can:
provide a non-productive themed version of my application using Carbon

#### Additional information
 - [Carbon theming project board](https://github.com/orgs/carbon-design-system/projects/33)
 - [Carbon issue 3502 containing documentation to refactor component tokens to support a type theme](https://github.com/carbon-design-system/carbon/pull/3502)

#### Acceptance criteria
 - [ ] Create expressive theme in the style package 

_Original issue: https://github.ibm.com/webstandards/digital-design/issues/1248_

**Comment by jeffchew** [comment]
_On Aug 12, Jeff-Chew commented:_
Based on timing it looks like this won't be ready for development in Sprint 16. Some documentation on the October release for Carbon:

https://github.com/carbon-design-system/carbon/wiki/Carbon-October-Release-Draft-Notes
https://github.com/carbon-design-system/carbon/wiki/Carbon-Release-Schedule---2019

We may be able to start this if an early version (release candidate) is available, need to discuss with Carbon team on timing.

cc: @linda-carotenuto @hahnrob @Wonil-Suh1

**Comment by jeffchew** [comment]
@photodow can you provide any additional details around this story?

**Comment by photodow** [comment]
@jeffchew is there anything specific you'd like to know?

The latest news is this work is delayed due to Carbon still working on the solution on their end. Matt reached out earlier last week to @jeffchew and I to let us know they might be able to pull us in allowing us to help them implement Carbon's part as early as this week (as of writing this I haven't looped back around on this).

Here is a link to their project board: https://github.com/orgs/carbon-design-system/projects/33

Here is a quick example Josh put together to demonstrate their solution.
https://www.sassmeister.com/gist/a98e19c89210d91cdc2b6b923082b8a3

Basically:
- Ability to emit custom properties on any selector
- Ability to emit only changed custom properties on any selector relative to parent theme
- Token values themselves are updated to a CSS Custom Property

**Comment by RobertaJHahn** [comment]
@photodow did you loop back with Carbon?  Could you provide the latest status in this issue?

cc:  @ljcarot

**Comment by jeffchew** [comment]
@RobertaJHahn moving this to sprint 22 after catching up with @wonilsuhibm on a recent design meeting with Carbon yesterday.
cc: @oliviaflory @photodow

**Comment by jeffchew** [comment]
Added dependency to #712 after various discussions, will need to approach the foundation story first before assessing the next steps for this story.

**Comment by jeffchew** [comment]
@RobertaJHahn @ljcarot @wonilsuhibm I think this story can be closed since we have already moved forward with the proxy approach, right?

**Comment by ljcarot** [comment]
@jeffchew yes, we've changed direction and instead of 'adding Carbon Marketing Theme', we're building the theme through our proxy approach.

**Comment by RobertaJHahn** [comment]
Closing this issue as the implementation direction has changed.
